setup.py

```diff
@@ -1,37 +1,50 @@
+#!/usr/bin/env python
+# -*- coding: utf-8; mode: python -*-
 
-import os
-import sys
+import re
+import codecs
 from setuptools import setup, find_packages
 
-here = os.path.abspath(os.path.dirname(__file__))
+def read_file(fname):
+    with codecs.open(fname, 'r', 'utf-8') as f:
+        return f.read()
 
-with open(os.path.join(here, 'README.rst')) as f:
-    README = f.read()
+def find_meta(meta):
+    """
+    Extract __*meta*__ from META_FILE.
+    """
+    meta_match = re.search(
+        r"^__{meta}__\s*=\s*['\"]([^'\"]*)['\"]".format(meta=meta),
+        META_FILE
+        , re.M )
+    if meta_match:
+        return meta_match.group(1)
+    raise RuntimeError("Unable to find __{meta}__ string.".format(meta=meta))
 
-with open(os.path.join(here, 'CHANGES.txt')) as f:
-    CHANGES = f.read()
+NAME             = 'tokenlib'
+REQUIRES         = []
+EXTRAS_REQUIRE   = {}
+META_FILE        = read_file('tokenlib/__init__.py')
+LONG_DESCRIPTION = [ read_file(n) for n in ['README.rst', 'CHANGES.txt']]
 
-requires = []
-if sys.version_info < (2, 7):
-    requires.append('unittest2')  # pragma: nocover
+setup(name                   = NAME
+      , version              = find_meta('version')
```

Comment:
Did we decide to put these commas on the end of the line rather than the start, for consistency with the broader codebase?

Reply:
Mea culpa. I was inattentive, sorry. I will fix it.